doc/examples/particles/web/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Usually we ignore the platform files on the examples, like ios and android folder, could this be ignored as well so we avoid having too much files on the examples?

Reply:
Yup, just missed that web project files weren't in the `.gitgnore` yet and i was doing scaffolding for the example from `dev` flutter channel, this should be gone now